cfgov/core/services.py

```diff
@@ -10,10 +12,17 @@
 from dateutil.parser import parse
 from v1.forms import CalenderPDFFilterForm
 
+
+class PDFReactorNotConfigured(Exception):
+    pass
+
 ## TODO: Update to python 3 when PDFreactor's python wrapper supports it.
 if six.PY2:
-    from core.lib.PDFreactor import *
-
+    try:
+        sys.path.append(os.environ.get('PDFREACTOR_LIB'))
```

Comment:
Does that work with the default https://github.com/cfpb/cfgov-refresh/pull/1075/files#diff-ac27c5919c46548eada7a7388e726c59R156 

I think sys.path entries need to be directories. (so, the 'lib' directory containing PDFReactor.py)